src/Methods.php

```diff
@@ -2057,9 +2080,9 @@ public function listNexus($filter, $top, $skip, $orderBy)
      * @param string $line2 The first address line portion of this address.
      * @param string $line3 The first address line portion of this address.
      * @param string $city The city portion of this address.
-     * @param string $region The region, state, or province code portion of this address.
+     * @param string $region Name or ISO 3166 code identifying the region portion of the address.      This field supports many different region identifiers:   * Two and three character ISO 3166 region codes   * Fully spelled out names of the region in ISO supported languages   * Common alternative spellings for many regions      For a full list of all supported codes and names, please see the Definitions API `ListRegions`.
```

Comment:
@ted-spence-avalara some wierd new line stuff happening here

Reply:
Got it, we're improving that in the 18.1 release.  For the moment, is that a blocker?